js/bybit.js

```diff
@@ -4265,93 +4890,224 @@ module.exports = class bybit extends Exchange {
         //         "time": "1666768215157"
         //     }
         //
+        const result = this.safeValue (response, 'result', {});
+        const trades = this.safeValue (result, 'list', []);
+        return this.parseTrades (trades, market, since, limit);
+    }
+
+    async fetchMyDerivativesTrades (symbol = undefined, since = undefined, limit = undefined, params = {}) {
```

Comment:
this name seems ambiguous, can we change it to something like `fetchMyUnifiedTrades` or `fetchMyUnifiedDerivativeTrades` ?

Reply:
How about we use `fetchMyUnifiedMarginTrades`? Maybe we can also change `fetchMyContractTrades` to `fetchMyDerivativesTrades`.

Reply:
@sc0Vu sounds good to me